onnxruntime/core/common/threadpool.cc

```diff
@@ -623,7 +626,7 @@ int ThreadPool::DegreeOfParallelism(const concurrency::ThreadPool* tp) {
   // When not using OpenMP, we parallelise over the N threads created by the pool
   // tp, plus 1 for the thread entering a loop.
   if (tp) {
-    if (CPUIDInfo::GetCPUIDInfo().IsHybrid()) {
+    if (tp->force_hybrid_ || CPUIDInfo::GetCPUIDInfo().IsHybrid()) {
```

Comment:
How is access to private member allowed here?

Reply:
Yes it is allowed.

Reply:
DegreeOfParallelism could access private members of an instance.